apps/nextjs-website/src/components/organisms/ChatbotLayout/ChatbotLayout.tsx

```diff
@@ -68,36 +71,43 @@ const ChatbotLayout = ({
           vertical: 'bottom',
           horizontal: 'right',
         }}
+        marginThreshold={0}
         disableScrollLock
         slotProps={{
           paper: {
             sx: {
               backgroundColor: 'transparent',
-              borderRadius: 3,
-              width: '40%',
-              minWidth: '48rem',
+              borderRadius: { xs: 0, md: 3 },
+              width: { xs: '100%', md: '40%' },
+              height: { xs: '100%', md: '40rem' },
```

Comment:
![image](https://github.com/user-attachments/assets/d24674a1-7568-430e-b897-7544ab5bca5b)
For small resolution desktop this height could breake the UI and not permit to send messages.
I think this should solve the problem, so for small resoliton display make the box relative to the view port and make input always accesible:
```suggestion
              height: { xs: '100%', md: '70vh' },
```